compiler/src/dmd/statement.h

```diff
@@ -725,8 +725,8 @@ class InlineAsmStatement final : public AsmStatement
 {
 public:
     void *asmcode;
-    unsigned asmalign;          // alignment of this statement
-    unsigned regs;              // mask of registers modified (must match regm_t in back end)
+    unsigned long long regs;      // mask of registers modified (must match regm_t in back end)
```

Comment:
just double checking that `unsigned long long` is actually what is desired here for a 64 bit integer.